<!--  -->

<launch>
	<node pkg="mylocalization" name="mylocalization_gridmap" type="mylocalization_gridmap" output="screen" >
		<param name="points_topic" value="lidar_points" />
		<param name="grid_resolution" value="0.02" />
		<param name="grid_height" value="2500" />
		<param name="grid_width" value="2500" />
	</node>
	
	<node pkg="rviz" name="rviz" type="rviz" args="-d $(find mylocalization)/rviz/mylocalization_gridmap.rviz" />
</launch>
